### DISCUSSION TITLE
freeradius-server link mismatch

### DISCUSSION THREAD

**Discussion by avramd** [Homebrew/discussions#5290] [Everyday usage] [answered]
### Output of `brew config`

```shell
➜  ~ brew config
HOMEBREW_VERSION: 4.2.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9edabf02ce29e54d0b717baa757cff38ad0c0e25
Last commit: 2 days ago
Core tap JSON: 15 Apr 15:00 UTC
Core cask tap JSON: 15 Apr 15:00 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.44.0 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 13.6.6-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
➜  ~
```

### Output of `brew doctor`

```shell
➜  ~ brew doctor
Your system is ready to brew.
➜  ~
```

### Description of issue

`freeradius-server` seems to install with a linking mismatch. When I run it, I get:

```
➜  ~ sudo radiusd -X
Password:
libssl version mismatch.  built: 30100030 linked: 30200010
➜  ~ echo $?
1
➜  ~ 
```

This still happens after running `brew update`. I've also uninstalled & reinstalled `freeradius-server`

### Additional Information
MacOSX version: 13.6.6 (22G630)
```
Darwin AGD-VoltServer-MBA-96524.local 22.6.0 Darwin Kernel Version 22.6.0: Mon Feb 19 19:43:41 PST 2024; root:xnu-8796.141.3.704.6~1/RELEASE_ARM64_T8103 arm64
```

**Comment by SMillerDev** [comment]
Does it actually fail? Or does it just complain?

**Reply by avramd** [reply]
It exits immediately. Sorry I forgot to show the following shell prompt, I've edited the OP to reflect this. 

For reference to anyone reading this, -X implies -s, which is "single server mode," and does not fork. So if the command exits, the server is not running.

**Reply by avramd** [reply]
Also, the exit code is `1`.

**Comment by chenrui333** [comment, marked as answer]
should be fixed with https://github.com/Homebrew/homebrew-core/pull/169198

**Reply by avramd** [reply]
Wow, that was fast - thank you so much!
